pom.xml

```diff
@@ -1081,6 +1081,15 @@ under the License.
 				</executions>
 			</plugin>
 
+			<!-- Pull bundled transitive dependencies (e.g. Mini-KDC see DIRSHARED-134) -->
```

Comment:
Would change "e.g." to "i.e." because it is the only dependency which uses the bundle mechanism.